packages/nodes-base/nodes/Airtop/actions/file/File.resource.ts

```diff
@@ -0,0 +1,45 @@
+import type { INodeProperties } from 'n8n-workflow';
+
+import * as deleteFile from './delete.operation';
+import * as get from './get.operation';
+import * as getMany from './getMany.operation';
+
+export { deleteFile, get, getMany };
+
+export const description: INodeProperties[] = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		displayOptions: {
+			show: {
+				resource: ['file'],
+			},
+		},
+		options: [
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Get a details of an uploaded file',
+				action: 'Get a file',
+			},
+			{
+				name: 'Get Many',
+				value: 'getMany',
+				description: 'Get a list of uploaded files',
```

Comment:
Let's avoid confusion that this might get you a list of downloadable files and just get the details for them. Maybe the following?
```suggestion
				description: 'Get details of multiple uploaded files',
```